### ISSUE TITLE
Feature Request: Special paste on text entries (text transformation)

### ISSUE THREAD

**Issue by VeRychard** [PasteBar/PasteBarApp#120]
It would be great to have some basic string manipulation before pasting text entries.
It's a ditto feature as well, many ideas could be adopted from it.

![353575706-6b4dfcae-8e75-46c5-a854-955e2c436f51](https://github.com/user-attachments/assets/3764ed4c-d1c4-434b-b827-22dd27caa9fe)

**Comment by kurdin** [comment]
Yeah, I was thinking about that as well. It should have some predefined ways/utils to transform text like UPPER CASE, lower case, etc but also be flexible enough to allow user to add its own string manipulation methods using JavaScript language. 

Lets keep this issue open and I'll think about implementation. I am busy right now with another project but I'll work in it in the future. This is useful.

**Comment by kurdin** [comment]
@VeRychard 
implemented with #275
